learn/getting-started/writing-a-grpc-service-with-ballerina.md

```diff
@@ -87,55 +108,66 @@ $ bal grpc --mode service --input helloworld.proto --output greeter_service/
 Once successfully executed, you will see the output below.
 
 ```bash
+Continuing with the existing protoc executor file at /var/folders/6p/8h4k83hj0r98n1s6mc3jy0480000gn/T/protoc-3.9.1-osx-x86_64.exe
 Successfully extracted the library files.
 Successfully generated the Ballerina file.
 ```
 
-Once you move into the directory, you will see the following two files,
+This creates two files below inside the `greeter_service` directory.
 
 ```bash
-greeter_service
-├── - - -
+.
+├── Ballerina.toml
 ├── greeter_service.bal
 └── helloworld_pb.bal
 ```
 
-Here,
-- The `helloworld_pb.bal` file is the stub file. It contains classes that the client/service uses to talk to each other
-and the Ballerina types corresponding to the request and response messages.
-- The `greeter_service.bal` file is the service template file. It contains service(s) with all the remote methods
+- The `helloworld_pb.bal` file is the stub file, which contains classes that the client/service uses to talk to each other and the Ballerina types corresponding to the request and response messages.
+- The `greeter_service.bal` file is the service template file, which contains service(s) with all the remote methods
   defined in the `.proto` file.
 
-#### Update and run the application
+### Updating the service template file
 
-Let’s update the service template file (i.e., `greeter_service.bal`) and add the business logic to the remote method. In this case, you only need to update the `sayHello` method as shown below.
+To add the business logic to the remote method (in this case, you only need to update the `sayHello` method as shown below), follow these steps:
 
-```ballerina
-import ballerina/grpc;
+1. Open the `greeter_service` directory in your text editor. 
 
-listener grpc:Listener grpcListener = new (9090);
+    >**Tip:** If you are using VS Code, in the terminal, navigate to the , and execute the `code .` command.
 
-@grpc:ServiceDescriptor {descriptor: ROOT_DESCRIPTOR_HELLOWORLD, descMap: getDescriptorMapHelloworld()}
-service "Greeter" on grpcListener {
+2. Replace the service template file (i.e., `greeter_service.bal`) with the code below.
 
-   remote function sayHello(HelloRequest value) returns HelloReply|error {
-       return { message: "Hello " + value.name};
-   }
-}
-```
+    ```ballerina
+    import ballerina/grpc;
+
+    listener grpc:Listener grpcListener = new (9090);
+
+    @grpc:ServiceDescriptor {descriptor: ROOT_DESCRIPTOR_HELLOWORLD, descMap: getDescriptorMapHelloworld()}
+    service "Greeter" on grpcListener {
+
+      remote function sayHello(HelloRequest value) returns HelloReply|error {
+          return { message: "Hello " + value.name};
+      }
+    }
```

Comment:
Code is  not properly formatted. Havign an extra space at the beginning.

Reply:
Fixed.